perun-core/src/test/java/cz/metacentrum/perun/core/impl/CacheManagerTest.java

```diff
@@ -149,7 +149,7 @@ public void getAllNonEmptyAttributesByStartPartOfName() throws Exception {
 		List<Attribute> attrs = cacheManager.getAllAttributesByStartPartOfName("urn:perun:group:attribute-def:opt", groupHolder);
 
 		assertTrue("result should contain group attribute", attrs.contains(groupAttr));
-		assertTrue("result should contain attribute definition", attrs.contains(attrDef));
```

Comment:
Actually, you can just suppress this warning. This is a problem with inheritance and we know about it. It should work properly. You can also add a comment explaining why it can be suppressed.